ophyd/areadetector/plugins.py

```diff
@@ -646,15 +646,12 @@ def set(self, region):
             Any of the keywords can be ommitted, and they will be ignored.
         '''
         if region is not None:
-            status = None
+            status = []
             for direction, value in region.items():
-                if status == None:
-                    status = getattr(self, 'min_xyz.min_{}'.format(direction)).put(value[0])
-                else:
-                    status = status & getattr(self, 'min_xyz.min_{}'.format(direction)).put(value[0])
-                status = status & getattr(self, 'size.{}'.format(direction)).put(value[1])
+                status.append( getattr(self, 'min_xyz.min_{}'.format(direction)).put(value[0]) )
```

Comment:
The extra space after `append(` breaks PEP8. I imagine you put it there in a reasonable effort to make this long line easier to parse. Another way to do that, and reduce repetition, would be:

```py
status0 = getattr(___).set(value[0])
status1 = getattr(___).set(value[1])
statuses.extend([status0, status1])
```

Also remember the earlier comment that `put` needs to be changed to `set`. "Put" is the low-level "Just send the EPICS command," which is ultimately at the bottom of "set" as well, but "set" adds additional machinery to track done-ness.